admin-ui/src/pages/request-detail-page.tsx

```
@@ -171,10 +174,10 @@ export function RequestDetailPage(): React.JSX.Element {
               <span className="font-mono text-sm">{item.request_id}</span>
               <StatusBadge status={item.http_status} />
               {item.duration_ms != null ? (
-                <Badge variant="outline">dur_ms: {fmtNum(item.duration_ms)}</Badge>
+                <Badge variant="outline">{t("requestDetailPage.badges.durMs")}: {fmtNum(item.duration_ms)}</Badge>
               ) : null}
               {item.ttfb_ms != null ? (
```

ESLint

[error] Expected '!==' and instead saw '!='.

(eqeqeq)